config/README.md

```diff
@@ -0,0 +1,8 @@
+Secrets generation:
+
+- Base64 secret was generated with `echo 'quarkus-qe-base64' | base64` command
+- SHA256 secret was generated with `echo -n "quarkus-qe-sha256" | openssl dgst -sha256` command
+- AES/GCM/NO-PADDING was generated with this method https://github.com/quarkiverse/quarkus-file-vault/blob/main/file-vault-common/src/main/java/io/quarkiverse/filevault/util/EncryptionUtil.java#L52
+
+Keystore was created with `keytool -importpass -alias config.property.name -keystore keystore -storepass quarkuspwd -storetype PKCS12 -v`
```

Comment:
Is keystore validity a concern for us?

Reply:
When Keystore is not valid, build fails. I actually experienced it several times thanks to mu clumsiness. I think this validation failure scenario belongs to upstream rather than to our integration tests. I won't pretend I found in SmallRye Config Keystore Source such test :-) but loading is handled by standard library (`java.security.KeyStore#load(java.io.InputStream, char[])`), so I am not concerned about keystores.